format-specs/geoparquet.md

```diff
@@ -50,57 +50,100 @@ Version of the geoparquet spec used, currently 0.3.0
+        }
+    },
+    "coordinate_system": {
+        "subtype": "ellipsoidal",
+        "axis": [
+        {
+            "name": "Geodetic longitude",
+            "abbreviation": "Lon",
+            "direction": "east",
+            "unit": "degree"
+        },
+        {
+            "name": "Geodetic latitude",
+            "abbreviation": "Lat",
+            "direction": "north",
+            "unit": "degree"
+        }
+        ]
+    },
+    "id": {
+        "authority": "OGC",
+        "code": "CRS84"
+    }
+}
 ```
 
 Due to the large number of CRSes available and the difficulty of implementing all of them, we expect that a number of implementations will start without support for the optional `crs` field.
-Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. But data that is better served in particular projections can choose to use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen crs.
+Users are recommended to store their data in longitude, latitude (OGC:CRS84 or not including the `crs` field) for it to work with the widest number of tools. Data that are more appropriately represented in particular projections may use an alternate coordinate reference system. We expect many tools will support alternate CRSes, but encourage users to check to ensure their chosen tool supports their chosen CRS.
+
+
+For implementations that operate entirely with longitude, latitude coordinates
+and are not CRS-aware or do not have easy access to CRS-aware libraries that can
+fully parse PROJJSON, it may be possible to infer that coordinates conform to
+the OGC:CRS84 CRS based on elements of the `crs` field.  For simplicity, Javascript
+object dot notation is used to refer to nested elements.
+
+The CRS is likely equivalent to OGC:CRS84 for a GeoParquet file if the `id` element is present:
+
+* `id.authority` = `"OGC"` and `id.code` = `"CRS84"`
+* `id.authority` = `"EPSG"` and `id.code` = `4326` (due to longitude, latitude ordering in this specification)
+
+The CRS is likely equivalent to OGR:CRS84 if all of the following are true:
+* `"type"` = `"GeographicCRS"`
+* `coordinate_system.axis[0].unit` = `"degree"`
+* `coordinate_system.axis[1].unit` = `"degree"`
+* the values for `coordinate_system.axis[n].direction` are `"east"` and `"north"` (in either order, for n in [0,1])
+
+and at least one of the following are true:
+* `datum.id.authority` = `"EPSG"` and `datum.id.code` = `6326`
+* `datum_ensemble.id.authority` = `"EPSG"` and `datum_ensemble.id.code` = `6326`
+* `datum_ensemble.ellipsoid.semi_major_axis` = `6378137` and `datum_ensemble.ellipsoid.inverse_flattening` = `298.257223563`
+* `datum.ellipsoid.semi_major_axis` = `6378137` and `datum.ellipsoid.inverse_flattening` = `298.257223563`
```

Comment:
The "checks" that are described here are for checking quite exact equivalence with EPSG:4326 or OGC:CRS84 (using the exact same datum definition), but based on https://github.com/opengeospatial/geoparquet/discussions/98 that might not necessarily be needed?